### PR TITLE
Fix missing default import from ethers

### PR DESCRIPTION
**Pull request by coreyar**
Ethers doesn't have a default export

### CODE REVIEW

File: src/utilities/common.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import * as constants from 'utilities/constants';
 import BigNumber from 'bignumber.js';
 import { getVaiTokenAddress } from './addressHelpers';
-import ethers from 'ethers';
+import * as ethers from 'ethers';
```
Comment (baomastr) [inline src/utilities/common.ts]: import {ethers} from 'ethers';

Comment (baomastr) [reply src/utilities/common.ts]: it was already fixed earlier

Comment (coreyar) [reply src/utilities/common.ts]: These are equivalent for this package but I am happy to use a named import

**Comment by baomastr** [comment]
resolved in https://github.com/VenusProtocol/venus-protocol-interface/pull/94

**Pull request closed without merging**